1-raw-data/gcs/gcs.go

```diff
@@ -30,10 +30,12 @@ import (
 	"io"
 	"log"
 	"net/http"
+	"strings"
 )
 
 type Connection interface {
-	StoreInBucket(ctx context.Context, fileName, url string) error
+	StoreTextContentInBucket(ctx context.Context, fileName, text string) (err error)
```

Comment:
Is there a reason you're naming the return error?

Reply:
Same in the following return statements.

Reply:
Its an approach that helped resolve certain functions where I needed to declare and create an empty value for the return variables before capturing the values and it simplifies returning multiple variables in the function. I decided to go for a consistent style across the code vs only one offs.